examples/device/device.cpp

```diff
@@ -77,6 +81,24 @@ struct CXX_QDMI_Operation_impl_d {
   std::string name;
 };
 
+struct CXX_QDMI_TelemetrySensor_impl_d {
+  std::string id;
+  std::string unit;
+  std::chrono::seconds sampling_rate;
+};
+
+struct CXX_QDMI_Device_TelemetrySensor_Query_impl_d {
+  std::chrono::time_point<std::chrono::system_clock> start_time;
+  std::chrono::time_point<std::chrono::system_clock> end_time;
+  size_t timeout{};
+  CXX_QDMI_TelemetrySensor telemetry_sensor{};
+  std::vector<std::chrono::time_point<std::chrono::system_clock>>
+      result_timestamps;
+  std::vector<float> result_values;
+  size_t result_length{};
+  QDMI_TelemetrySensor_Query_Status status{};
```

Comment:
```suggestion
  size_t result_length;
  QDMI_TelemetrySensor_Query_Status status;
```
If I am not mistaken, if you want to initialize those members, you need to assign values, i.e., `0` for the length and some enum value for the status. However, I think it should be just fine without `{}`.

Reply:
Even though they need `{}`, clang-tidy raises a [cppcoreguidelines-pro-type-member-init](https://clang.llvm.org/extra/clang-tidy/checks/cppcoreguidelines/pro-type-member-init.html) warning.

Reply:
✅ Done